Apps/frontend/src/shader/indicatorShader.js

```diff
@@ -0,0 +1,31 @@
+export const fragmentShader = `#version 300 es
+precision mediump float;
+
+in vec4 v_colour;
+out vec4 o_colour;
+
+void main() {
+  o_colour = v_colour;
+}
+`;
+export const vertexShader = `#version 300 es
+precision mediump float;
+
+uniform vec4 u_colour;
+uniform float u_sample;
+uniform int u_channel;
+uniform float u_offset;
+uniform float u_scale;
+
+in vec2 inputVertex;
+out vec4 v_colour;
+
+void main() {
+  float index = float(u_channel);
```

Comment:
Does `index` have to be `float` so we can compute with it down below (`float`s only work with `float`s)?

Reply:
Yes, WebGL doesn't cast types implicitly.